qoffer-infrastructure/src/main/groovy/life/qbic/PortletView.groovy

```diff
@@ -0,0 +1,15 @@
+package life.qbic
+
+/**
+ * Handles the ViewModel shown to the user of q-offer 2.0
+ *
+ * This class is responsible for the representation of the internal data
+ * in the form of individual portlet components presented to the user
```

Comment:
I am not sure about the definition of `ViewModel` here. Is this class expected to contain logic? As for our `sample-tracking-update-portlet` we did handle the logic in the view and its components and used the `ViewModel` only to store data and properties of the view.